corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -185,7 +185,14 @@ def test_process_doc_from_sql(self, datetime_mock):
 
 class StaticKafkaIndicatorPillowTest(TestCase):
 
-    @patch('corehq.apps.callcenter.data_source.get_call_center_domains', MagicMock(return_value=[domain_lite('cc1')]))
+    @patch(
+        'corehq.apps.userreports.pillow.'
+        'ConfigurableReportTableManagerMixin.get_all_configs',
+        MagicMock(return_value=[]))
+    @patch(
+        'corehq.apps.userreports.pillow.'
+        'ConfigurableReportTableManagerMixin.rebuild_tables_if_necessary',
+        MagicMock(return_value=None))
```

Comment:
Is it worth running this tests with this mocked out. Doesn't seem like it's testing anything that isn't tested elsewhere.